addons/mail/static/src/new/thread/message.xml

```diff
@@ -4,7 +4,7 @@
 <t t-name="mail.message" owl="1">
     <div class="o-mail-message py-1 mt-1 px-3 d-flex" t-att-class="{'o-mail-is-author': message.isAuthor}" t-ref="ref">
         <div class="o-mail-message-sidebar d-flex flex-shrink-0 justify-content-center">
-            <div t-if="!props.squashed" class="o-mail-avatar-container position-relative">
+            <div t-if="!props.squashed" class="o-mail-avatar-container position-relative" t-on-click="() => messaging.openChat({partnerId: message.authorId})">
                 <img class="w-100 h-100 rounded-circle o_object_fit_cover o_redirect" t-att-class="{ 'cursor-pointer': props.hasActions }"
```

Comment:
Should put `cursor-pointer` at the same level as the `t-on-click`. And this class should be conditionally added in the same way that this action results to `openChat` (so not clickable if author is current user, because it won't `openChat`).